IREC/IREC_PDD.tex

```diff
@@ -0,0 +1,383 @@
+% \nomenclature{$\dot{m}$}{Mass flow rate
+%   \nomunit{\,\si{\kilo\gram\per\second}}}
+% \nomenclature{$c$}{Speed of light
+%  \nomunit{\,\SI{2.9979e8}{\meter\per\second}}}
+\printnomenclature{}
+
+
+% HELPFUL HINT: If you get the warning ``Command terminated with space.'' when using a \command try placing ``%'' or ``{}'' immediately following the command.
+
+% The sections included here are required. Additional sections and subsections may be added as necessary.
+\section{Introduction}
+\label{sec:Introduction}
+  % The introduction is a place to give background and context before diving into the subject matter.
+  % Establish context for the work you are about to propose and the main ideas of the proposition itself.
+
+\IEEEPARstart{T}{he} Intercollegiate Rocket Engineering Competition is an annual event hosted by ESRA for student rocketry teams from across the USA and around the world.
+Beginning in 2017, IREC became the flagship activity of a new event called the Spaceport America Cup which is held annually in the desert of Las Cruces, New Mexico.
+The competing rockets are typically seen with a diameter of 4 to 8 inches, a length of anywhere from 8 to 20 feet, and will travel to an altitude of either 10,000 or 30,000
+feet depending on whether the competing team opts to enroll into the basic or advanced category. The Space Dynamics Laboratory sponsors a separate competition at the
+Spaceport America Cup called the SDL Payload Challenge where scientific payloads integrated into the IREC rockets are judged based on criteria such as scientific relevance and technical execution.
+
+The RIT Space Exploration team is a student-faculty research organization dedicated to giving students the opportunity to gain hands-on experience by working with space systems.
+By cooperating with Launch Initiative, SPEX would like to enter IREC in the basic category. LI will provide the rocket body and the rocket engine while SPEX will focus on the
+integrated scientific payload with the intent of competing in the SDL payload challenge.
+
+\section{Primary Objective}
+\label{sec:Primary Objective}
+  % At the end of the day, whether the project ``succeeds'' or ``fails'' is judged against the objectives it sought to meet.
+  % Note that results that contradict expectations/hypotheses are not failures if the scientific \& engineering methods are followed along the way.
+  % Sometimes our expectations are wrong and that can be just as successful as getting data we thought we'd see.
+  % What matters are what questions you intend to answer.
+  % This is the main purpose or main goal the project hopes to achieve.
+The primary objective of SPEX's involvement in this competition is to develop and deliver a functioning scientific payload that integrates with Launch Initiative's rocket
+structure and successfully performs in the June 2018 competition. Successfully meeting this primary objective implies
```

Comment:
I don't like placing hard dates in the PDD. In my view, this document outlines a clear vision for the project and its goals, but is independent of any one point in time. For example, one portion of the project takes 2 weeks, but the PDD should only state that duration, not the calendar weeks it will take place.

I say you give a hard look at whether this date is essential to the design or scientific objective of this project. I'm not convinced it is